channel-email/src/main/java/com/synopsys/integration/alert/channel/email/action/EmailGlobalCrudActions.java

```diff
@@ -37,36 +35,30 @@ public EmailGlobalCrudActions(AuthorizationManager authorizationManager, EmailGl
         this.validator = validator;
     }
 
-    public ActionResponse<EmailGlobalConfigModel> getOne(Long id) {
-        return configurationHelper.getOne(
-            // Compiler wants this cast as it does not understand the types being used
-            (Supplier<Optional<EmailGlobalConfigModel>>) () -> configurationAccessor.getConfiguration(id).map(DatabaseModelWrapper::getModel)
-        );
```

Comment:
Interesting, it looks like removing the extra types in the middle helped resolve this casting issue. Were you able to still save values as expected when running Alert?

Reply:
Yes. AlertPagedModel uses generics as does DatabaseModelWrapper.  AlertPagedModel doesn't know the type inside of DatabaseModelWrapper because it is using generics as well.  Now because the type of the object inside the AlertPagedModel doesn't use generics the correct type can be inferred.  So you no longer need the cast to avoid the compiler issue with generics.

Reply:
It works fine to save the information.